dist/bootstrap-clockpicker.js

```diff
@@ -453,19 +453,27 @@
 			this.isAppended = true;
 		}
 
-		// Get the time
-		var value = ((this.input.prop('value') || this.options['default'] || '') + '').split(':');
-		if (value[0] === 'now') {
+    var trimedValue = ((this.input.prop('value') || this.options['default'] || '') + '').replace(/^\s\s*/, '').replace(/\s\s*$/, '');
+
+		if (trimedValue === 'now') {
 			var now = new Date(+ new Date() + this.options.fromnow);
-			value = [
-				now.getHours(),
-				now.getMinutes()
-			];
-		}
-		this.hours = + value[0] || 0;
-		this.minutes = + value[1] || 0;
+      this.hours = now.getHours();
```

Comment:
There is an small indent issue. Mb fix the indent for the whole file in a specific commit ?